examples/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "removeComments": false,
     "sourceMap": false,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es2017"
+    "target": "es2017",
+    "typeRoots": ["./node_modules/@types"]
```

Comment:
I think this is not needed, can you try?

Reply:
You're absolutely right, it was a remnant of my first attempt with a dummy typing for `noble-mac`.